### ISSUE TITLE
Why does this project have a setting file

### ISSUE THREAD

**Issue by alonisser** [hasadna/knesset-data-django#5]
If this is a reusable django app (as the presumption in the README) it is supposed to be using a settings.py file especially with all of oknesset original apps and supporting apps, etc
This isn't how reusable apps are supposed to work. It's supposed to expose settings to the using project and perhaps get apps from using projects.

**Comment by OriHoch** [comment]
sorry.for the confusion.. I remember that I clarified this somewhere but maybe not clear enough.. 

this project is not a standard standalone django module. it has dependency on open knesset and also open knesset has dependency on it

this is meant to be a temporary situation due to limitations of our goals and the complexity of the open knesset code

TLDR
the end goal is to have a standalone django module but we are not there yet..

I'll keep this issue open to ensure I update the documentation

**Comment by OriHoch** [comment]
in any case, feel free to suggest improvements if you think the code and architecture can be improved.. it's very likely there is a better solution..

**Comment by OriHoch** [comment]
also, when used from open knesset - the open knesset settings file is used, not the one in knesset-data-django

**Comment by OriHoch** [comment]
documented this in known issues section on README with link to this issue
